algo

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I think I would prefer adding this prompt to Ansible and removing the shell commands from our script that dash does not support. Then we can keep it at /bin/sh.